data/campaigns/The_Rise_Of_Wesnoth/scenarios/14_Rough_Landing.cfg

```diff
@@ -41,8 +42,8 @@
         name= _ "Abraxas"
         side=2
         canrecruit=yes
-        {GOLD 90 120 150}
-        {INCOME 2 3 4}
+        {GOLD 80 115 150}
+        {INCOME 1 3 5}
```

Comment:
I already reduced the gold values a lot so please keep the income the same as I had it for all sides. I'm really not sure you need to touch the values in this scenario actually, did you try playing it?

Reply:
I played thru 1.14 with similar changes; attaching my replay 
[TRoW-Rough Landing replay.gz](https://github.com/wesnoth/wesnoth/files/6133747/TRoW-Rough.Landing.replay.gz)